feature/kanban_board/models/project.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const projectSchema = new Schema({
+    title: {
+        type: String,
+        required: true
+    },
+    columns: [{
+        type: Schema.Types.ObjectId,
+        ref: 'Column'
+    }]
+});
+
+module.exports = mongoose.model('Project' , projectSchema);
```

Comment:
make sure changes handled in a different PR are not reflected in subsequent pull-requests.

Reply:
resolved